### ISSUE TITLE
Configure CustomFormatter exceptions as object in appsettings

### ISSUE THREAD

**Issue by ronnyek** [serilog/serilog-sinks-elasticsearch#118]
I notice you have some examples of how to configure elastic stuff via the appsettings, and that stuff in itself does work pretty flawless.

As the exceptions appear to be basically unfilterable and dont display well, I want to allow configuration via appsettings.json, and am not aware if I can actually tell it to use the custom formatter or not. Am I missing something simple?

**Comment by NeilCross** [comment]
Figured i'd leave a comment here for anyone else facing the same issue.

There's no option via appsettings to provide the properties for the CustomFormatter, as it only loads via a type name. I worked around this by inheriting from the formatter I needed with a new class with an empty constructor and loaded that type instead.

![image](https://user-images.githubusercontent.com/1518421/60074641-c4757080-9766-11e9-84ac-b2bb31b49044.png)


Then configured that type via the standard mechanism:
![image](https://user-images.githubusercontent.com/1518421/60074523-6ba5d800-9766-11e9-8f55-3e2879033ed9.png)
